packages/client/katello-pull-provider-migrate/katello-pull-provider-migrate.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+
+# set SYSCONFDIR to /etc if it is not set
+if [ -z "$SYSCONFDIR" ]; then
+    SYSCONFDIR=/etc
+fi
+
+# see if /etc/rhsm/rhsm.conf exists
+RHSMCONF_PATH=$SYSCONFDIR/rhsm/rhsm.conf
+if [ -f $RHSMCONF_PATH ]; then
```

Comment:
What if we removed RHSMCONF_PATH and just relied on the output of subscription-manager now that reliance on rhsm.conf itself was removed?